### PR TITLE
Feature Flags on Flutter SDK

### PR DESCRIPTION
**Pull request by davidbielik**

### CODE REVIEW

**Comment by davidbielik** [comment]
@jerielng @hokstuff i could use your help to review the code samples in detail 👨‍🎨 . Feel free to make edits yourself if that's easier!

File: _docs/_developer_guide/platform_wide/feature_flags/create.md [hunk 1]
```diff
@@ -386,6 +408,14 @@ for(const feature of features) {
 }
 ```
 {% endtab %}
+{% tab Flutter %}
+```dart
+List<BrazeFeatureFlag> featureFlags = await braze.getAllFeatureFlags();
+featureFlags.forEach((feature) {
+  print("Feature: " + feature.id + " " + feature.enabled);
```
Comment (jerielng) [inline _docs/_developer_guide/platform_wide/feature_flags/create.md:415]: I think we should be able to interpolate the string like this:

```suggestion
  print("Feature: ${feature.id} ${feature.enabled}");
```

Comment (davidbielik) [reply _docs/_developer_guide/platform_wide/feature_flags/create.md:415]: nice just like javascript without needing `\`` backticks

File: _docs/_developer_guide/platform_wide/feature_flags/create.md [hunk 2]
```diff
@@ -550,6 +585,19 @@ BrazePlugin.subscribeToFeatureFlagUpdates((featureFlags) => {
 });
 ```
 {% endtab %}
+{% tab Flutter %}
+```dart
+// Create stream subscription
+StreamSubscription featureFlagsStreamSubscription;
+
+featureFlagsStreamSubscription = braze.subscribeToFeatureFlagsUpdatedEvent(List<BrazeFeatureFlag> featureFlags) {
+  print("feature flags were updated")
```
Comment (jerielng) [inline _docs/_developer_guide/platform_wide/feature_flags/create.md:594]:
```suggestion
  print("Feature flags were updated");
```

File: _docs/_developer_guide/platform_wide/feature_flags/create.md [hunk 3]
```diff
@@ -550,6 +585,19 @@ BrazePlugin.subscribeToFeatureFlagUpdates((featureFlags) => {
 });
 ```
 {% endtab %}
+{% tab Flutter %}
+```dart
+// Create stream subscription
+StreamSubscription featureFlagsStreamSubscription;
+
+featureFlagsStreamSubscription = braze.subscribeToFeatureFlagsUpdatedEvent(List<BrazeFeatureFlag> featureFlags) {
```
Comment (jerielng) [inline _docs/_developer_guide/platform_wide/feature_flags/create.md:593]:
```suggestion
featureFlagsStreamSubscription = braze.subscribeToFeatureFlags((featureFlags) {
```

File: _docs/_developer_guide/platform_wide/feature_flags/create.md [hunk 4]
```diff
@@ -550,6 +585,19 @@ BrazePlugin.subscribeToFeatureFlagUpdates((featureFlags) => {
...
+  print("feature flags were updated")
+}
```
Comment (jerielng) [inline _docs/_developer_guide/platform_wide/feature_flags/create.md:595]:
```suggestion
});
```

**Review by josh-mccrowell-braze — APPROVED** [review]
You're burning through these like a champion

**Pull request merged**